apps/olat-api/static/activityTypes.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "id": "LIVE_QUIZZES",
+    "title": "Live Quiz Overview",
+    "path": "/liveQuizzes",
+    "olatConfigurationKey": "live-quizzes",
+    "isSubselectionRequired": false,
+    "isEmailTransferRequired": false
+  },
+  {
+    "id": "PRACTICE_QUIZZES",
+    "title": "Practice Quiz Overview",
+    "path": "/practiceQuizzes",
+    "olatConfigurationKey": "practice-quizzes",
+    "isSubselectionRequired": false,
+    "isEmailTransferRequired": false
+  },
+  {
+    "id": "MICRO_LEARNINGS",
+    "title": "Micro Learning Overview",
+    "path": "/microLearnings",
+    "olatConfigurationKey": "micro-learnings",
+    "isSubselectionRequired": false,
+    "isEmailTransferRequired": false
+  },
+  {
+    "id": "CREATE_ACCOUNT",
+    "title": "Create Account",
+    "path": "/createAccount",
+    "olatConfigurationKey": "create-account",
+    "isSubselectionRequired": false,
+    "isEmailTransferRequired": true
+  },
+  {
+    "id": "DOCS",
+    "title": "Documentation",
+    "path": "/docs",
+    "olatConfigurationKey": "docs",
+    "isSubselectionRequired": false,
+    "isEmailTransferRequired": false
+  },
+  {
+    "id": "LIVE_QUIZ",
+    "title": "Live Quiz",
+    "path": "/liveQuiz",
```

Comment:
for this to work, we need to add this page to the pages folder and add a redirect to PWA_URL/session/LIVE_QUIZ_ID